datafusion/core/src/logical_plan/builder.rs

```diff
@@ -52,7 +52,7 @@ pub const UNNAMED_TABLE: &str = "?table?";
 
 /// Builder for logical plans
 ///
-/// ```
+/// ``` ignore
```

Comment:
This will be addressed as part of https://github.com/apache/arrow-datafusion/issues/2308 once we move the builder into the `datafusion-expr` crate